src/AddTodoForm.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+// import * as reactDom from 'react-dom'
+import { useState } from "react";
+
+function AddTodoForm (props) {
+ const handleAddTodo = (event) => {
+        event.preventDefault();
+     const todoTitle = event.target.title.value;
+        console.log(todoTitle, `ARE YOU HERE111`);
+      const form = document.querySelector('form');
+      form.reset();
```

Comment:
While this works perfectly fine, another more direct approach that doesn't involve the creation of another variable is `event.target.reset()`